browser-profiler/dom-bridger/index.ts

```diff
@@ -46,6 +46,30 @@ export default class DomBridger {
     polyfill.remove = polyfill.remove.filter(x => !isVariationChange(x.path, x.propertyName));
     polyfill.add = polyfill.add.filter(x => !isVariationChange(x.path, x.propertyName));
   }
+
+  public static removeUnsupportedPropertiesFromPolyfill(polyfill: IDomPolyfill): void {
+    polyfill.modify = polyfill.modify.filter(
+      x => !isUnsupportedProperty(x.path, x.propertyName, x.property),
+    );
+    polyfill.remove = polyfill.remove.filter(
+      x => !isUnsupportedProperty(x.path, x.propertyName, null),
+    );
+    polyfill.add = polyfill.add.filter(
+      x => !isUnsupportedProperty(x.path, x.propertyName, x.property),
+    );
+  }
+}
+
+function isUnsupportedProperty(path: string, propertyName: string, property: any): boolean {
```

Comment:
Is this fine or should this be something else? Unsupported properties are props that are currently not supported by either profiler or emulator. 

This could also be split in two and stuff emulator cant handle can just be dropped there. This was easiest solution and is probably good enough for now?

Reply:
Fine for now